### DISCUSSION TITLE
Uji H0 dan H1 serta tren omset dari data historis

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#521] [Q&A] [answered]
Berdasarkan data set yg ada, yaitu kurang dari 2 tahun, jumlah bulan atau n < 30, apakah sebaiknya uji H0 dan H1 memakai Z test atau t tes ?  serta bgm melihat kemungkinan forecast dari tren tsb utk memperkuat hasil uji tes tsb ? Terimakasih

<img width="528" alt="image" src="https://user-images.githubusercontent.com/123436963/216226476-0fddd04e-7053-48aa-9951-c8b447e25370.png">

**Comment by kevbow** [comment, marked as answer]
Dear @Micoalgo,

Jika jumlah data kurang dari 30 maka sebaiknya pengujian hipotesis menggunakan t-test.
Sedangkan untuk forecast kondisi yang akan datang merupakan materi Time Series & Forecasting yang merupakan salah satu ranah machine learning.
Demikian yang dapat kami sampaikan, semoga membantu.
Jika jawaban ini sesuai dengan pertanyaan Anda silakan pilih "Mark as answer". Terima kasih.

**Comment by Micoalgo** [comment]
Terimakasih jawabannnya
